packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -10,92 +10,137 @@ import {
   concatAST,
 } from 'graphql';
 import chalk from 'chalk';
-import {dirname} from 'path';
+import {dirname, isAbsolute, resolve} from 'path';
 import {readJSON, readFile, writeFile, mkdirp} from 'fs-extra';
 import {watch} from 'chokidar';
 import * as glob from 'glob';
-import {compile, Operation, Fragment, AST} from 'graphql-tool-utilities/ast';
+import {
+  getGraphQLConfig,
+  GraphQLProjectConfig,
+  GraphQLConfig,
+} from 'graphql-config';
+import {
+  compile,
+  isOperation,
+  Operation,
+  Fragment,
+  AST,
+} from 'graphql-tool-utilities/ast';
+import {
+  Headers as FetchHeaders,
+  RequestInit as FetchRequestInit,
+} from 'node-fetch';
 
 import {printDocument, printSchema} from './print';
 import {EnumFormat} from './types';
 
+// adding some type definitions to the global namespace so that graphql-request
```

Comment:
I just want to make sure — this doesn't end up conflicting with other TS projects consuming it (notably, Sewing Kit)?

Reply:
that's a good point. AFAIK, it should not because only the `dom` lib places these types in the global namespace. anything node related should be exported via the `node-fetch` module. The truth is that it _could_ conflict with anything that expects these types coming from the `dom` lib, but perhaps this *is* the appropriate response as we probably want to fix those usages anyways. That said, i'll 🎩 against SK right now and confirm.